newIDE/app/src/ResourcesList/ResourceExternalEditor.flow.js

```diff
@@ -15,9 +15,10 @@ export type ExternalEditorOpenOptions = {|
       path?: string,
       name: string,
       originalIndex: ?number,
-      metadata?: string,
+      externalEditorData?: string,
```

Comment:
Here we have externalEditorData which can either be a string (in case it's attached to a resource, or an Object, in case it's attached to all the changes.

It's a bit confusing I think. Can't we always have an object? I'm not sure why the externalEditorData would be a string?

Reply:
fixed :)